### ISSUE TITLE
​refreshFromDisk may work for the wrong reason

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#3416]
See PR #3433.

From Félix:

​In ​​refreshFromDisk, near the very beginning of that method, there is a check to see if there's a filename, and that the filename is not a 'folder' instead of an actual file. 
```python
    if not fn:
        g.warning(f"not an @<file> node: {p.h!r}")
        return
    # #1603.
    if os.path.isdir(fn):
        g.warning(f"not a file: {fn!r}")
        return
```
When it's checking for the fn string to be a possible 'directory', it's checking with the filename only. eg. myFile.txt, which is a non-absolute path/string...  instead of what a call to c.fullPath would provide. e.g. C:\myFolder\myFile.txt. 

(which is what is used in leoAtFile.py's methods read, readOneAtAutoNode, readOneAtCleanNode, readOneAtEditNode when they want to have a full path string of the intended file to work with.)

So my concern is that the check in refreshFromDisk done with os.path.isdir(fn) always returns false, which still makes most every case unintentionally work fine, work for the wrong reason.

**Comment by tbpassin** [comment]
I think the check should be
```
if not os.path.isfile(full_file_path):
        g.warning(f"not a file: {full_file_path!r}")
        return
```

instead - using the full path, of course.

**Comment by edreamleo** [comment]
@boltex @tbpassin PR #3433 fixes two other quirps :-)

**Comment by edreamleo** [comment]
Closed via PR #3433.
